src/components/shared/About/privacy-policy/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,121 @@
+'use client';
+
+import { useTranslations } from 'next-intl';
+
+import styles from './PrivacyPolicy.module.css';
+
+const PrivacyPolicyModel = (props: any) => {
+  
+  const t = useTranslations();
+
+  const lastUpdated = '5/5/2024';
+  const emailAddress = 'mapofpi@gmail.com';
```

Comment:
Philip has the mapofpi domain..  would look more professional.
-> this is not a comment :)

Reply:
Agreed!  I was told about having some issues with the mapofpi domain so we're using mapofpi@gmail.com for the time being.  I'll need to revisit with him.